- riscv: test for assembler support
  b3825bbe452c8e4f129fa90bba1fed0ee7b87d71

Index: libswscale/riscv/Makefile
--- libswscale/riscv/Makefile.orig
+++ libswscale/riscv/Makefile
@@ -1,3 +1,3 @@
 OBJS += riscv/rgb2rgb.o
-OBJS += riscv/rgb2rgb_rvb.o
+RV-OBJS += riscv/rgb2rgb_rvb.o
 RVV-OBJS += riscv/rgb2rgb_rvv.o
